integration-tests/tests/docload/docload.spec.js

```diff
@@ -44,10 +44,13 @@ module.exports = {
     const docFetch = await browser.globals.findSpan(span => span.name === 'documentFetch');
     const scriptFetch = await browser.globals.findSpan(span => span.name === 'resourceFetch' && span.tags['http.url'].includes('splunk-rum.js'));
     const docLoad = await browser.globals.findSpan(span => span.name === 'documentLoad');
+
+    const brokenImgFetch = await browser.globals.findSpan(span => span.name === 'resourceFetch' && span.tags['http.url'].includes('/nosuchimage.jpg'));
```

Comment:
not an issue, but for my own knowledge: wouldn't you say that if you have an <img> in a page, then its failure to fetch is a child span of the docload span, and consequently it's part of the same trace?

Reply:
Great call, these all should have the same traceId - I'll add those asserts.